starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +50,21 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  /**
+   * Test Create method 
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    inventory.setId(ID);
+    Inventory actualInventoryItem = this.inventoryDAO.create(inventory);
+    Assert.assertNotNull(actualInventoryItem);
+    Assert.assertEquals(inventory.getProductType(), actualInventoryItem.getProductType());
+    Assert.assertEquals(inventory.getName(), actualInventoryItem.getName());
+    Assert.assertNotNull(inventory.getId());
```

Comment:
This `Assert.assertNotNull(inventory.getId()` is not necessary as the test guarantees an inventory saved with a generated id.
This line can be removed.

Reply:
Resolved by commit 9d5309298864b98643594aed7d9b9d1fe4d06239